### DISCUSSION TITLE
✅  Bought WebSSH pro couldn’t find in my App Store

### DISCUSSION THREAD

**Discussion by jayyangliu** [isontheline/pro.webssh.net#512] [Q&A]
I bought bought WebSSH pro in China App Store,I couldn’t find on my purchased app how can I get WebSSH pro back on my App Store so I can download it.

**Comment by isontheline** [comment]
I'm sorry but WebSSH isn't available in China App Store anymore.

https://github.com/isontheline/pro.webssh.net/discussions/227
https://github.com/isontheline/pro.webssh.net/discussions/249

**Reply by jayyangliu** [reply]
i do have US apple ID would you help me get my webssh pro i bought on us app store?

**Reply by jayyangliu** [reply]
any email i could contact you for further help?

**Reply by isontheline** [reply]
If you change your country store you will be required to purchase it again I think
https://mengus.net/#contact
